docs/references/astro/sign-up-store.mdx

```diff
@@ -5,34 +5,66 @@ description: Clerk's $signUpStore nanostore provides a convenient way to access
 
 # `$signUpStore`
 
-The `$signUpStore` store provides a convenient way to access the [`SignUp`](/docs/references/javascript/sign-up/sign-up){{ target: '_blank' }} object, which allows you to check the current state of a sign-up. This is also useful for creating a custom sign-up flow.
+The `$signUpStore` store provides a convenient way to access the [`SignUp`](https://clerk.com/docs/references/javascript/sign-up/sign-up) object, which allows you to check the current state of a sign-up. This is also useful for creating a custom sign-up flow.
 
 ## How to use the `$signUpStore` store
 
 ### Check the current state of a sign-up
 
 The following example demonstrates how to use the `$signUpStore` store to access the `SignUp` object and check the current state of a sign-up.
 
-```tsx {{ filename: 'sign-up-step.tsx' }}
-import { useStore } from '@nanostores/react'
-import { $signUpStore } from '@clerk/astro/client'
+<CodeBlockTabs options={['React', 'Vue', 'Svelte']}>
+  ```tsx {{ prettier: false, filename: 'sign-up-step.tsx' }}
```

Comment:
Same question about formatting the React code block

```suggestion
  ```tsx {{ filename: 'sign-up-step.tsx' }}
```